docs/csi_ug_uninstall_cli.md

```diff
@@ -0,0 +1,20 @@
+# Uninstalling the driver with GitHub
+
+Use this information to uninstall the IBM® CSI (Container Storage Interface) operator and driver using  from a command line terminal.
```

Comment:
```suggestion
Use this information to uninstall the IBM® CSI (Container Storage Interface) operator and driver using from a command line terminal.
```

Reply:
why is the "and" here in red?

Reply:
I assume GitHub thinks this is code, so it highlights common code syntax, as I noticed [happens with `exec`](https://github.com/IBM/ibm-block-csi-driver/blob/2b61376d8398ad2aad110d4e327b4c23f3073706/README.md#driver-usage) (in blue)

Reply:
@rivkap Why the conversation is resolved?

Reply:
see the latest content - it was updated

Reply:
[This](https://github.com/IBM/ibm-block-csi-driver/blob/277e0ba1e6b0b4a26ff52ab2a158bdffa3a7c74e/docs/content/installation/csi_ug_uninstall_cli.md) is the latest, no?
Do we want to keep "using from"?

Reply:
@zingero correct file is content/installation/csi_ug_uninstall_github.md

Reply:
ok